app/assets/stylesheets/partials/_lists.scss

```diff
@@ -6,3 +6,18 @@
 .notification-error-explanation:only-child {
   list-style: none;
 }
+
+ul.dashed {
+  list-style: none;
+  padding-left: 0;
+}
+
+ul.dashed > li {
```

Comment:
Are you sure we would like have direct Child here? Also usage of tag in selector? 

Maybe adding class nam on that `li` tag